app.js

```diff
@@ -3,16 +3,18 @@ import express from 'express'
 import nodeCleanup from 'node-cleanup'
 import routes from './routes.js'
 import { init, cleanup } from './whatsapp.js'
+import cors from 'cors'
 
 const app = express()
 const host = process.env.HOST ?? '127.0.0.1'
-const port = parseInt(process.env.PORT ?? 8000)
+const port = parseInt(process.env.PORT ?? 3333)
 
+app.use(cors())
 app.use(express.urlencoded({ extended: true }))
 app.use(express.json())
 app.use('/', routes)
 
-app.listen(port, host, () => {
+app.listen(port, () => {
```

Comment:
Also don't remove this host part. I think you can just set `0.0.0.0` for the `HOST` key from `env` file instead of removing this part.

Reply:
i tried but on heroku its not working

Reply:
also see cors package 📦,i used it so that api can be assessed from any domain without any error

https://crunchify.com/how-to-fix-access-control-allow-origin-issue-for-your-https-enabled-wordpress-site-and-maxcdn/

Reply:
> Also don't remove this host part. I think you can just set `0.0.0.0` for the `HOST` key from `env` file instead of removing this part.

i will try that

Reply:
> also see cors package 📦,i used it so that api can be assessed from any domain without any error
> 
> https://crunchify.com/how-to-fix-access-control-allow-origin-issue-for-your-https-enabled-wordpress-site-and-maxcdn/

but implement this
https://www.section.io/engineering-education/how-to-use-cors-in-nodejs-with-express/

Reply:
> i tried but on heroku its not working

I see, i'm just gonna make the host part optional then.

Reply:
> also see cors package 📦,i used it so that api can be assessed from any domain without any error
> 
> https://crunchify.com/how-to-fix-access-control-allow-origin-issue-for-your-https-enabled-wordpress-site-and-maxcdn/

Yeah i'm okay with the cors.